python/Jets_Setup_cff.py

```diff
@@ -38,7 +38,7 @@ def setup_jets(process, cms, options, postfix="PFlow"):
     if usePrivateSQlite:
         from CondCore.DBCommon.CondDBSetup_cfi import *
         import os
-        era = "Summer15_50nsV4_"
+        era = "Summer15_50nsV4_" #now version 5
```

Comment:
will do - but it is of little importance anyway for the moment as these are for the JEC files.